index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Example HTML</title>
+</head>
+<body>
+    <h1>This is just a test.</h1>
+    <p>Obviously</p>
+</body>
+</html>
```

Comment:
There should be a newline at the end of document in accordance with style guidelines.